style.css

```
@@ -10,31 +10,349 @@
   font-optical-sizing: auto;
   font-weight: 700;
   font-style: normal;
+
 }
 
 html {
-
+  width: 1479px;
+  margin: 0;
+  padding: 0;
   box-sizing: border-box;
 }
 
 body {
+  width: 1479px;
   margin: 0;
+  padding: 0;
   font-family: 'Montserrat', sans-serif;
 }
 
+
+.header {
+  font-family: 'Montserrat', sans-serif;
+  position: fixed;
+  width: 100%;
+  background-size: cover;
+  background-position: center;
+
+}
+
+.hero-title {
+  width: 270px;
+  margin-top: 594px;
+  margin-left: 70px;
+  font-size: 24px;
+  font-weight: 400;
+  line-height: 31px;
+  color: aliceblue;
+  margin-bottom: 16px;
+}
+
 .container {
-  max-width: 1100px;
+  max-width: 1300px;
   margin: 0 auto;
 }
 
-.header {
-  background-image: url('./images/mainbike.jpg');
-  width: 1441px;
-  height: 1002px;
+.header__title {
+
+  display: flex;
+  margin-top: 35px;
+  justify-content: space-between;
+  align-items: center;
+  height: 90px;
 }
 
-.header__title {
+.logo--streetster {
+  width: 161px;
+  height: 65px;
+}
+
+.header__search {
+  flex-grow: 1;
+  display: flex;
+  justify-content: center;
+}
+
+.header__search input {
+  width: 400px;
+  height: 30px;
+  border-radius: 15.5px;
+  border: none;
+  padding-left: 40px;
+  background-image: url('images/icons/.svg');
+  background-size: 20px;
+  background-position: 10px center;
+  background-repeat: no-repeat;
+}
+
+.header__icons {
+  display: flex;
+  align-items: center;
+  gap: 20px;
+}
+
+.logo--cart {
+  width: 26px;
+  height: 25px;
+  filter: invert(1);
+}
+
+.burger {
+  width: 26px;
+  height: 20px;
   display: flex;
+  flex-direction: column;
   justify-content: space-between;
-  align-items: end;
+  background: none;
+  border: none;
+  cursor: pointer;
+}
+
+.burger__line {
+  width: 26px;
+  height: 1.82px;
+  background-color: white;
+}
+
+.hero {
+  font-family: 'Montserrat', sans-serif;
+  min-height: 1003px;
+  width: 100%;
+  background-image: url('images/mainbike.jpg');
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+
+  padding: 20px;
+}
+
+.hero-title {
+  width: 270px;
+  margin-top: 594px;
+  margin-left: 70px;
+  font-size: 24px;
+  font-weight: 400;
+  line-height: 31px;
+  color: aliceblue;
+  margin-bottom: 16px;
+}
+
+.hero-subtitle {
+  margin-top: 0;
+  color: aliceblue;
+  font-size: 64px;
+  width: 1120px;
+  font-weight: 700;
+  line-height: 77px;
+  margin-left: 70px;
+  margin-bottom: 0;
+
+}
+
+.hero-button {
+  width: 206px;
+  height: 41px;
+  margin-top: 20px;
+  margin-left: 70px;
+  border-width: 3px;
+  border-style: solid;
+  border-color: white;
```

Такие вещи можно сокращать

```suggestion
  border: 3px solid white;
```